cli/generate/src/rules.rs

```diff
@@ -441,8 +441,8 @@ impl TokenSet {
     }
 
     pub fn len(&self) -> usize {
-        self.eof as usize
-            + self.end_of_nonterminal_extra as usize
+        usize::from(self.eof)
+            + usize::from(self.end_of_nonterminal_extra)
```

Comment:
not a fan of these, `as _` reads a bit nicer

Reply:
sure, will revert, but do note that here the order of precedence starts to play a role, i.e. `as` vs `+` -- so parens might be more explicit. `T::from(...)` imo is more explicit and simpler in that sense, but you are the maintainer :)

Reply:
It is more explicit, but I find the cognitive load of reading it a bit higher, I take an extra second to look at it, vs an `as _` being, to me, "oh yeah we're just casting a number type"